kaldb/src/main/proto/kaldb_configs.proto

```diff
@@ -85,6 +85,11 @@ message QueryServiceConfig {
   string managerConnectString = 3;
 }
 
+enum KafkaOffsetLocation {
+  BEGINNING = 0;
```

Comment:
Here's another place where we introduce more terms describing where the offset is at. To recap:

Kafka configs use `earliest` and `latest`, which is read into `beginningOffset` and `endOffset` respectively.
Kaldb configs use `beginning` and `current`, which is read into `tailOffset` and `headOffset` respectively. 

We should consolidate these to use the Kafka pattern for describing these.